plugins/catalog/src/data/utils.tsx

```diff
@@ -13,8 +13,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import React from 'react';
 import { Component } from './component';
 import { Entity, Location } from '@backstage/catalog-model';
+import Edit from '@material-ui/icons/Edit';
+import IconButton from '@material-ui/core/IconButton';
+import { styled } from '@material-ui/core/styles';
+
+const DescriptionWrapper = styled('span')({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const createEditLink = (url: string): string => url.replace('blob', 'edit');
```

Comment:
tricky if username or repo have 'blob' as a part of the string :)

Reply:
True, this is veeeeery shaky. Creating a new PR with a slightly improved one.